understanding/22/accessible-authentication-minimum.html

```diff
@@ -68,11 +68,11 @@ <h2>In brief</h2>
     <section id="intent">
         <h2>Intent of Accessible Authentication</h2>
 
-        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method to log in. Most Web sites rely on usernames and passwords for logging in. Memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
+        <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method for users to authenticate -- to prove they are an authentic account holder. Most Web sites rely on usernames and passwords for logging in. Memorizing or transcribing a username, password, or one-time verification code places a very high or impossible burden upon people with certain cognitive disabilities.</p>
 
         <p>While Web sites can use the recognition of objects or of non-text content provided by the user to meet this Success Criterion, such techniques do not fully support the cognitive accessibility community and should be avoided if possible. Refer to <a href="accessible-authentication-enhanced">Accessible Authentication (Enhanced)</a> for guidance to be more inclusive and accessible.</p>
 
-        <p>This Success Criterion does <em>not</em> address creation of a username or initiating an account.  For many web sites, establishing an initial username is not significantly different than logging in with that username.  However, for many other web sites, the processes are quite different.  Moreover, the means used for creating an account vary more than the means used for re-authentication.  This is because initial authentication may require more due diligence with assuring the identity of a new user.  Another substantive difference between first authentication and re-authentication is that a third party, human or automated, is often required.</p>
+        <p>This Success Criterion does <em>not</em> address creation of a username or initiating an account.  For many Web sites, establishing an initial username is not significantly different than logging in with that username.  However, for many other Web sites, the processes are quite different.  Moreover, the means used for creating an account vary more than the means used for authentication.  This is because initial validation may require more due diligence with assuring the identity of a new user.  Another substantive difference between first account creation and authentication is that a third party, human or automated, is often required in the former.</p>
```

Comment:
different than > different from

overall though, the argumentation here feels a bit spurious though, even after the update